dotnet/src/SemanticKernel/CoreSkills/TextMemorySkill.cs

```diff
@@ -96,35 +96,31 @@ public async Task<string> RetrieveAsync(SKContext context)
     /// SKContext["input"] = "what is the capital of France?"
     /// {{memory.recall $input }} => "Paris"
     /// </example>
-    /// <param name="text">The input text to find related memories for</param>
-    /// <param name="context">Contains the 'collection' to search for the topic and 'relevance' score</param>
-    [SKFunction("Semantic search and return up to N memories related to the input text")]
-    [SKFunctionName("Recall")]
-    [SKFunctionInput(Description = "The input text to find related memories for")]
-    [SKFunctionContextParameter(Name = CollectionParam, Description = "Memories collection to search", DefaultValue = DefaultCollection)]
-    [SKFunctionContextParameter(Name = RelevanceParam, Description = "The relevance score, from 0.0 to 1.0, where 1.0 means perfect match",
-        DefaultValue = DefaultRelevance)]
-    [SKFunctionContextParameter(Name = LimitParam, Description = "The maximum number of relevant memories to recall", DefaultValue = DefaultLimit)]
-    public async Task<string> RecallAsync(string text, SKContext context)
+    /// <param name="text">The input text to find related memories for.</param>
+    /// <param name="collection">Memories collection to search.</param>
+    /// <param name="relevance">The relevance score, from 0.0 to 1.0, where 1.0 means perfect match.</param>
+    /// <param name="limit">The maximum number of relevant memories to recall.</param>
+    /// <param name="context">Contains the memory to search.</param>
+    [SKFunction, Description("Semantic search and return up to N memories related to the input text")]
+    public async Task<string> RecallAsync(
+        [SKName("input"), Description("The input text to find related memories for")] string text,
+        [Description("Memories collection to search"), DefaultValue(DefaultCollection)] string? collection,
```

Comment:
The default value should actually be `this._collection` if it Additionally, this logic was missed in other methods in #705 (my bad).